### PR TITLE
docs: SECRET_KEY Rotation Documentation

### PR DESCRIPTION
**Pull request by sujiplr**
Additional documentation for SECRET_KEY rotation and SECRET_KEY setting up.

### SUMMARY
Currently the SECRET_KEY rotation and SECRET_KEY setting up is missing in the documentation. So added the steps for covering the same.

### CODE REVIEW

File: docs/docs/installation/running-on-kubernetes.mdx [hunk 1]
```diff
@@ -92,6 +92,34 @@ postgresql:
   postgresqlPassword: superset
 ```
 
+Make sure you are giving your own SECRET_KEY for the encryption instead of default SECRET_KEY.
```
Comment (dpgaspar) [inline docs/docs/installation/running-on-kubernetes.mdx:95]: Consider replacing by:
"""
Make sure, you set a unique strong complex alphanumeric string for your `SECRET_KEY` and use a tool to help you generate a sufficiently random sequence
"""

**Review by dpgaspar — COMMENTED** [review]
Left a couple of comments, thank you for your contribution!

File: docs/docs/installation/configuring-superset.mdx [hunk 2]
```diff
@@ -20,7 +20,10 @@ ROW_LIMIT = 5000
 SUPERSET_WEBSERVER_PORT = 8088
 
 # Flask App Builder configuration
-# Your App secret key
+# Your App secret key, this will be used for encrypting the data.
+# Make sure you are changing this key for your deployment with a strong key.
+# You can generate a strong key using `openssl rand -base64 42`
+
 SECRET_KEY = '\2\1thisismyscretkey\1\2\e\y\y\h'
```
[the remark this answers is not in the document]
Comment (sujiplr) [reply docs/docs/installation/configuring-superset.mdx:27]: Done.

File: docs/docs/installation/running-on-kubernetes.mdx (hunk 1 above)
Comment (sujiplr) [reply docs/docs/installation/running-on-kubernetes.mdx:95]: Done.

**Review by dpgaspar — APPROVED** [review]

**Comment by dpgaspar** [comment]
LGTM is it ready to go @sujiplr ?

**Comment by sujiplr** [comment]
> LGTM is it ready to go @sujiplr ?

Yes.

**Review by villebro — APPROVED** [review]
Thanks for this, I'm sure this will be very helpful to lots of people!

**Review by dpgaspar — APPROVED** [review]

**Pull request merged**